File: mobile/src/main/java/com/chernowii/camcontrol/camera/goproAPI/model/media/GoProFolder.java
package com.chernowii.camcontrol.camera.goproAPI.model.media;

import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;
import java.util.List;

public class GoProFolder {

    @SerializedName("d")
    public String folder;

    @SerializedName("fs")
    public List<GoProFile> data = new ArrayList();
}

File: mobile/src/main/java/com/chernowii/camcontrol/camera/goproAPI/model/media/GoProFolder.kt
package com.chernowii.camcontrol.camera.goproAPI.model.media

import com.google.gson.annotations.SerializedName

import java.util.ArrayList

class GoProFolder {

    @SerializedName("d")
    var folder: String? = null

    @SerializedName("fs")
    var data: List<GoProFile> = ArrayList()
}
